src/Illuminate/Foundation/Auth/ThrottlesLogins.php

```diff
@@ -51,7 +50,7 @@ protected function sendLockoutResponse(Request $request)
         );
 
         throw ValidationException::withMessages([
-            $this->username() => [Lang::get('auth.throttle', ['seconds' => $seconds])],
+            $this->username() => [trans('auth.throttle', ['seconds' => $seconds])],
```

Comment:
IMHO: the less helper in the framework code, the better

Reply:
I've made a search across all framework and there were no other usages of Lang facade. There are several usages of trans() helper, though.